packages/arch/packages/tooltip/src/index.js

```diff
@@ -172,7 +177,7 @@ export default class Tooltip extends Component<Props, State> {
   };
 
   render() {
-    const { children, content, onHide, onShow, placement, className } = this.props;
+    const { children, content, onHide, onShow, placement, modifiers, className } = this.props;
```

Comment:
I'd prefer not to extend the tooltip's API. I think, in this case, we're better off just applying the applicable modifiers:

```jsx
<Popper
  modifiers={{ preventOverflow: { boundariesElement: 'scrollParent' } }}
/>
```

The user shouldn't have to know about Popper's API to have the tooltip work as expected in an overflow container.